app/views/log_events/index.html.haml

```diff
@@ -0,0 +1,52 @@
+= content_for :h1 do
+  = text("pages.global_settings")
+
+= content_for :sidebar do
+  = render("admin/shared/sidenav_global", sidenav_tab: "log_events")
+
+%h2= text("pages.event_log")
+
+= form_tag(log_events_path, method: :get) do
+
+  = label_tag(text(:event_filter))
+  = select_tag(:events,
+    options_for_select([[text("account__create"), "account__create"],
+                        [text("account__update"), "account__update"],
+                        [text("account_user__create"), "account_user__create"],
+                        [text("account_user__delete"), "account_user__delete"],
+                        [text("user__create"), "user__create"]], params[:events]),
+    multiple: true,
+    class: "js--chosen",
+    data: { placeholder: text(:event_placeholder) })
+
+  = label_tag(text(:start_date))
+  = text_field_tag(:start_date, params[:start_date], class: :datepicker)
```

Comment:
We have a `datepicker__data` class which will get loaded no matter what page it's on. It has a nice feature of being able to set min/max dates for the picker via data attributes (we don't necessarily need that here).